var/spack/repos/builtin/packages/pumi/package.py

```diff
@@ -45,10 +45,25 @@ class Pumi(CMakePackage):
 
     variant('zoltan', default=False, description='Enable Zoltan Features')
     variant('fortran', default=False, description='Enable FORTRAN interface')
+    variant('simmodsuite', default='none',
+        values=('none', 'base', 'kernels', 'full'),
+        description='''Enable Simmetrix SimModSuite Support: \'base\' enables
+        the minimum set of functionality, \'kernels\' adds CAD kernel support
+        to \'base\', and \'full\' enables all functionality''')
```

Comment:
This will add additional spaces in the middle of the string. You can confirm this if you run `spack info pumi`. Also, you can use `"` for string delimiters if you need to use `'` inside of the string. Then you don't need any backslashes.